universe/common/helper/worlds.go

```diff
@@ -72,6 +76,21 @@ func AddWorldFromTemplate(worldTemplate *WorldTemplate, updateDB bool) (uuid.UUI
 		}
 	}
 
+	// adding random space
+	rand.Seed(time.Now().Unix())
```

Comment:
it's not a cheap operation
you need to check is it really necessary, if yes, then move it to `main.go`